src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/blocking/CarBlocking_Manufacturer.java

```diff
@@ -0,0 +1,44 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.blocking;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.matching.blockers.generators.RecordBlockingKeyGenerator;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.Matchable;
+import de.uni_mannheim.informatik.dws.winter.model.Pair;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.processing.DataIterator;
+import de.uni_mannheim.informatik.dws.winter.processing.Processable;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CarBlocking_Manufacturer extends RecordBlockingKeyGenerator<Car, Attribute> {
```

Comment:
Basically the same as `CarBlockingKeyByManufacturerGenerator.java`. Could you just migrate the list of car brands and delete this?